### DISCUSSION TITLE
[Bug] Unable to send alert to Enterprise WeChat

### DISCUSSION THREAD

**Discussion by Seven-Wu-F** [apache/skywalking#12917] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

oap version is 9.7.0
My alarm-settings.yml file is configured as follows
![image](https://github.com/user-attachments/assets/e561abfa-7074-423c-9988-4d91dd91aa17)

### What you expected to happen

The oap service starts normally, and i can see the alarm generated on the UI, but WeChat does not receive it.
![image](https://github.com/user-attachments/assets/8ca95755-1a3b-4944-9453-e66ed3e793bc)

Is it my alarm-settings.yml file that's wrong? help pl！

### How to reproduce

Is it my alarm-settings.yml file that's wrong? help pl！

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-alarm/#wechat

We don't have WeChat account to support. But it is just a super direct and simple http request. I think you should debug that hook codes and generated http. It should be easy.

**Comment by Seven-Wu-F** [comment]
Just modify it to the configuration like this,working now,thanks for your help
```
  service_access_frequency_rule:
    expression: sum(service_cpm) >=1
    period: 1
    silence-period: 5
    message: Service instance {name} has been accessed more than once in the last minute.
    hooks:
      - "wechat.default"
hooks:
  wechat:
    default:
      text-template: |-
        {
          "msgtype": "text",
          "text": {
            "content": "Apache SkyWalking Alarm: \n %s."
          }
        }
      webhooks:
      - https://qyapi.weixin.qq.com/cgi-bin/webhook/send?key=
```

**Reply by wu-sheng** [reply]
9.7 is a little old, I am not sure whether default hook worked before
